src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -608,7 +608,57 @@ private void CheckNewModifier(Symbol symbol, bool isNew, DiagnosticBag diagnosti
 
                     if ((object)associatedPropertyOrEvent == null)
                     {
-                        diagnostics.Add(ErrorCode.ERR_OverrideNotExpected, overridingMemberLocation, overridingMember);
+                        bool suppressError = false;
+                        if (overridingMemberIsMethod)
+                        {
+                            foreach (var parameterType in ((MethodSymbol) overridingMember).ParameterTypes)
+                            {
+                                if (IsOrContainsErrorType(parameterType.TypeSymbol))
+                                {
+                                    suppressError = true; // The parameter type must be fixed before the override can be found, so suppress error
+                                    break;
+                                }
+                            }
+                            
+                        }
+                        if(!suppressError)
+                        {
+                            diagnostics.Add(ErrorCode.ERR_OverrideNotExpected, overridingMemberLocation, overridingMember);
+                        }
+
+                        bool IsOrContainsErrorType(TypeSymbol symbol)
+                        {
+                            if (symbol.IsErrorType())
+                                return true;
```

Comment:
There are multiple cases that changed in this method. But only one test was updated.
Please add tests to cover the remaining changes.

Reply:
> Is UseSiteErrorTests the correct place to add the tests? or somewhere else?

The "use site errors" term is normally used to refer to errors returned by Symbol.GetUseSiteDiagnostic API. It doesn't look like this scenario is about them. 

---
In reply to: [242983943](https://github.com/dotnet/roslyn/pull/31937#discussion_r242983943) [](ancestors = 242983943)

Reply:
I see a lot of tests about ```ErrorCode.ERR_OverrideNotExpected``` in src\Compilers\CSharp\Test\Semantic\Semantics\InheritanceBindingTests.cs

---
In reply to: [243014444](https://github.com/dotnet/roslyn/pull/31937#discussion_r243014444) [](ancestors = 243014444,242983943)

Reply:
Looking at the test files that reference `ERR_OverrideNotExpected`, `InheritanceBindingTests` stands out. That seems a good place to add such tests.

Reply:
Thanks. I will do so when I get a chance.